c/test/cpp/generate_sbp_data.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
Do we need to be able to regenerate the binary file?

Reply:
Yes, it's helpful to have the json version (and use it to generate binary) if we ever want to change/add tests.